runtime/queries/php/highlights.scm

```diff
@@ -8,6 +8,14 @@
 (named_type (name) @type) @type
 (named_type (qualified_name) @type) @type
 
+(namespace_definition
+  name: (namespace_name(name)) @namespace
+)
+
+(subscript_expression
+  (variable_name(name) @constant.builtin)
+)
```

Comment:
for code like

```php
$array = ['One', 'Two', 'Three'];
echo $array[0]; // => "One"
```

This now highlights `arrays` in L2 as `constant.builtin` but I think the `variable` capture from before was more accurate

Reply:
```
(subscript_expression
  (variable_name(name) @constant.builtin
    (#match? @constant.builtin "^_?[A-Z][A-Z\\d_]+$")
  )
)
```
it‘s seem ok